_Streams/113-return-neural-networks.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "The Return of Neural Networks"
+video_number: 113
+date: 2018-01-05
+video_id: raFdpMUWO9Q
+
+topics:
+  - title: "Using ES6 syntax with previous matrix math videos"
+    time: "29:30"
+    url: "https://www.youtube.com/watch?v=puWqNBFDMMk"
+  - title: "Matrix Math: Multiply"
+    time: "59:10"
+    url: "https://www.youtube.com/watch?v=NgZAIkDcPkI"
+  - title: "Matrix Math: Transpose"
+    time: "1:50:06"
+    url: "https://www.youtube.com/watch?v=nAz9AA_pQUQ"
+  - title: "Langton's ant Coding Challenge"
+    time: "2:08:00"
+    url: "/CodingChallenges/89-langtonsant"
+ 
+links:
+  - title: "The Coding Train on Amazon"
+    url: "https://www.amazon.com/shop/thecodingtrain"
+  - title: "Deeplearn.js"
+    url: "https://deeplearnjs.org/"
+  - title: "Repository for Experimentation with p5 and deeplearn.js"
+    url: "https://github.com/ITPNYU/ml5"
+  - title: "Hadamard product (matrices) on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Hadamard_product_(matrices)"
+  - title: "Matrix multiplication on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Matrix_multiplication"
+  - title: "Langton's Ant on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Langton%27s_ant"
+  - title: "ITP Physical Computing"
+    url: "https://itp.nyu.edu/physcomp/"
+  - title: "Bryan WC Chung's Pro Processing for Images and Computer Vision with OpenCV"
+    url: "https://www.amazon.com/Processing-Images-Computer-Vision-OpenCV/dp/1484227743"
```

Comment:
This is actually a book. Please refer to the [Content Contribution Guide](https://github.com/CodingTrain/website/wiki/Content-Contribution-Guide) on how to add books.